### DISCUSSION TITLE
Extend commonProps with "menuIsOpen".

### DISCUSSION THREAD

**Discussion by igoryusha22** [JedWatson/react-select#5296] [Ideas]
Sometimes we need to have different styles for opened and closed select, so we need to have an information about open state of select when we styling our select.

So let's extend `commonProps` with `menuIsOpen` so every custom will have it.

**Comment by aaronkeisler** [comment]
Has anyone found a way around this? I'd like to style the valueContainer differently when the menu is open vs when it is closed.

**Comment by aaronkeisler** [comment]
Wanted to bump this comment - any thoughts on a workaround here?
